tests/manual/classic.js

```diff
@@ -12,8 +12,8 @@ let editor, editable, observer;
 
 function initEditor() {
 	ClassicEditor.create( document.querySelector( '#editor' ), {
-		features: [ 'enter', 'typing', 'paragraph', 'undo', 'heading', 'basic-styles/bold', 'basic-styles/italic' ],
-		toolbar: [ 'headings', 'bold', 'italic', 'undo', 'redo' ]
+		features: [ 'enter', 'typing', 'paragraph', 'undo', 'heading', 'basic-styles/bold', 'basic-styles/italic', 'link' ],
```

Comment:
I don't think we should extend this test. It's for classic implementation, not for testing features. It's up to link's MT to see to that.